gemrb/core/Store.cpp

```diff
@@ -136,7 +136,7 @@ int Store::AcceptableItemType(ieDword type, ieDword invflags, bool pc) const
 		}
 
 		//...the shop is special (iwd/how/bg1)
-		if ((core->HasFeature(GFFlags::NO_UNDROPPABLE)) || (core->HasFeature(GFFlags::BREAKABLE_WEAPONS))) {
+		if ((core->config.GameType == "how") || (core->config.GameType == "bg1")) {
```

Comment:
No, like I said in the issue, the proper way is to add another game flag.

Reply:
I've added a new game flag and performed a simple test.

[cinnamon-2023-07-13T164020-0230.webm](https://github.com/gemrb/gemrb/assets/137374871/a41e6ec0-c9e9-461d-b2a5-208a61dab0b4)